src/pages/profile/index.js

```diff
@@ -15,16 +15,25 @@ const ViewProfile = () => {
   const navigate = useNavigate();
   const [ userId, setUserId ] = useState()
 	const { token } = useAuth();
+  const urlEndpoint = process.env.REACT_APP_API_URL
```

Comment:
Where is this variable needed?

Reply:
It's not needed at the minute and has been removed